modules/roles_profiles/manifests/profiles/gecko_t_osx_1014_generic_worker_staging.pp

```diff
@@ -51,23 +48,25 @@
                 version => '1.3.8',
             }
 
-            class { 'generic_worker':
+            class { 'generic_worker::multiuser':
                 taskcluster_client_id     => $taskcluster_client_id,
                 taskcluster_access_token  => $taskcluster_access_token,
-                livelog_secret            => $livelog_secret,
                 worker_group              => $worker_group,
                 worker_type               => $worker_type,
                 quarantine_client_id      => $quarantine_client_id,
                 quarantine_access_token   => $quarantine_access_token,
                 bugzilla_api_key          => $bugzilla_api_key,
-                generic_worker_version    => 'v13.0.3',
-                generic_worker_sha256     => '6e5c1543fb3c333ca783d0a5c4e557b2b5438aada4bc23dc02402682ae4e245e',
+                task_dir                  => '/Users',
+                generic_worker_version    => 'v16.4.0',
```

Comment:
I don't understand why you haven't hit [bug 1592233](https://bugzilla.mozilla.org/show_bug.cgi?id=1592233).